htdocs/install/mysql/migration/4.0.0-5.0.0.sql

```diff
@@ -24,6 +24,8 @@
 -- -- VPGSQL8.2 DELETE FROM llx_usergroup_user      WHERE fk_user      NOT IN (SELECT rowid from llx_user);
 -- -- VMYSQL4.1 DELETE FROM llx_usergroup_user      WHERE fk_usergroup NOT IN (SELECT rowid from llx_usergroup);
 
+-- after changing const name, please insure that old constant was rename
+UPDATE llx_const SET name = 'THIRDPARTY_DEFAULT_CREATE_CONTACT' WHERE name='MAIN_THIRPARTY_CREATION_INDIVIDUAL'
```

Comment:
hello, you update from 
MAIN_THIRPARTY_CREATION_INDIVIDUAL
is it not
MAIN_THIR**D**PARTY_CREATION_INDIVIDUAL
with D ?